draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -664,17 +681,20 @@ Packet as described in {{lorawan-schc-payload}}.
   * Unicast downlinks: ACK-Always.
   * Multicast downlinks: No-ACK, reliability has to be ensured by the upper
     layer. This feature is OPTIONAL and may not be implemented by SCHC gateway.
-* RuleID: 8 bits stored in LoRaWAN FPort.
-* Window index (unicast only): encoded on W=1 bit, as per [RFC8724].
-* DTag: Size is 0 bit, not used.
+* RuleID: 8 bits stored in LoRaWAN FPort. cf {{rule-id-management}}
+* DTag: Size T=0 bit, not used. cf {{DTag}}
 * FCN: The FCN field is encoded on N=1 bit, so WINDOW_SIZE = 1 tile.
-* RCS: Use recommended calculation algorithm in [RFC8724].
-* MAX_ACK_REQUESTS: 8.
-* Retransmission timer:  See {{downlink-retransmission-timer}}.
+* RCS: Use recommended calculation algorithm in [RFC8724], §8.2.3. Integrity Checking.
```

Comment:
Same here, is §8.2.3 intended?

Reply:
Yes, it is the chapter of the RFC8724 where the RCS is recommended.
It was the advice from B. Kaduk, I slightly changed it to:

```
RCS: Use recommended calculation algorithm in [RFC8724] (§8.2.3. Integrity Checking).
```

Is it more explicit ?